src/raftstore/store/worker/split_check.rs

```diff
@@ -123,19 +123,26 @@ impl<'a> MergedIterator<'a> {
 /// Split checking task.
 pub struct Task {
     region: Region,
+    auto_check: bool,
```

Comment:
`auto_check = false` can mean a lot of things, I prefer to define an enum explicitly like:
```
enum SplitType {
    AutoSplit,
    HalfSplit,
}
```